common/src/models/domain/environmentUrls.ts

```diff
@@ -2,5 +2,9 @@ export class EnvironmentUrls {
   base: string;
   api: string;
   identity: string;
+  icons: string;
+  notifications: string;
   events: string;
+  webVault: string;
+  keyConnector: string;
```

Comment:
I'm guessing these are used individually by the clients such as desktop?

Reply:
Most of these are used in all clients, we have just been using an `any` type for handling urls instead of this model. I tried to refactor all that to use a model, and discovered that we already had one that wasn't being used and just brought it up to speed.

Reply:
Got it, that makes sense! I'm definitely a fan a making those more strongly typed